src/utils.tsx

```diff
@@ -0,0 +1,310 @@
+import React from 'react';
+import { IColumnProps } from './Column';
+
+interface IDefaultPropsContainer extends Function {
+  defaultProps: any;
+}
+
+const isReactElementOrElementType = (x: any): x is (React.ElementType | React.ReactElement) => {
+  return true;
+};
+
+const isDefaultPropsContainer = (x: any): x is IDefaultPropsContainer => {
+  return true;
+};
+
+export function renderElement(renderer: React.ElementType | React.ReactElement | IDefaultPropsContainer, props?: any) {
+  if (!renderer) {
+    return null;
+  }
+
+  if (React.isValidElement(renderer)) {
+    return React.cloneElement(renderer, props);
+  } else if (typeof renderer === 'function') {
+    // if (renderer.prototype && renderer.prototype.isReactComponent) {
+    if (isReactElementOrElementType(renderer)) {
```

Comment:
@kohakukun  PTAL at this part. 
(Don't know why it's all green. Your branch didn't have a utils.js i thought.)